gufe/tests/test_network.py

```diff
@@ -0,0 +1,98 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+import pytest
+import networkx as nx
+
+from gufe import AlchemicalNetwork, ChemicalSystem, Transformation
+
+from .test_protocol import DummyProtocol, DummyProtocolResult
+
+
+@pytest.fixture
+def benzene_variants_star_map(
+    benzene,
+    toluene,
+    phenol,
+    benzonitrile,
+    anisole,
+    benzaldehyde,
+    styrene,
+    prot_comp,
+    solv_comp,
+):
+
+    variants = [toluene, phenol, benzonitrile, anisole, benzaldehyde, styrene]
+
+    # define the solvent chemical systems and transformations between benzene and the others
+    solvated_ligands = {}
+    solvated_ligand_transformtions = {}
```

Comment:
here and everywhere else `transformtions` -> `transformations`? (i.e. do we really save anything from a single letter?)

Reply:
I changed this